accounts/models.py

```diff
@@ -1,3 +1,21 @@
 from django.db import models
+from django.contrib.auth.models import User
 
 # Create your models here.
+
+class UserBillingInfo(models.Model):
+    customer = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
Model removed. I only wanted the user to have one set of billing info but the way the models were originally set up every time an order was saved a new one would be created. Instead of this setup I am now just getting the user info from the most recent order info.